pygame/draw.py

```diff
@@ -356,3 +358,252 @@ def rect(surface, color, rect, width=0):
 
     points = ((l, t), (r, t), (r, b), (l, b))
     return polygon(surface, color, points, width)
+
+
+def circle(surface, color, pos, radius, width=0):
+    """pygame.draw.circle(Surface, color, pos, radius, width=0): return Rect
+
+       draw a circle around a point"""
+    if radius < 0:
+        raise ValueError("negative radius")
+    if width < 0:
+        raise ValueError("negative width")
+    if radius < width:
+        raise ValueError("width greater than radius")
+
+    if not width:
+        _fillellipse(surface, pos, radius, radius, color)
+    else:
+        # Pygame explicitly loops from 1 here, so we do the same.
+        for loop in range(0, width):
```

Comment:
I'm a bit confused by the "explicitly loops from 1 here" comment since `range(0, width) == [0, 1, 2, ...]`?